docker/controller/Dockerfile

```diff
@@ -20,6 +20,14 @@ RUN CGO_ENABLED=0 go build -o tyk-k8s .
 ######## Start a new stage from scratch #######
 FROM debian:buster-slim
 
+RUN apt-get update \
+ && apt-get upgrade -y \
+ && apt-get install -y --no-install-recommends \
+            curl ca-certificates apt-transport-https debian-archive-keyring gnupg \
```

Comment:
If all you actually want is to add the certificate authorities, `ca-certificates` should be enough.